### PR TITLE
Remove presence of flag_meanings and flag_values variable attributes …

### PR DESCRIPTION
**Pull request by ggalibert**
…as being an enough evidence of a variable being a quality control one. Replaced by the presence of a quality_control_conventions variable attribute.

### CODE REVIEW

File: compliance_checker/imos/util.py [hunk 1]
```diff
@@ -194,8 +194,7 @@ def find_quality_control_variables(dataset):
                 quality_control_variables.append(var)
                 continue
 
-        if getattr(var, 'flag_values', None) is not None or getattr(var,\
-                    'flag_meanings', None) is not None:
+        if getattr(var, 'quality_control_conventions', None) is not None:
```
Comment (mhidas) [inline compliance_checker/imos/util.py]: If we don't need to do anything with the value of the attribute, it's simpler to say

``` python
if hasattr(var, 'quality_control_conventions'):
```

**Comment by mhidas** [comment]
@ggalibert  Looks good. Any chance you could add a unit test for this util function?

**Comment by ggalibert** [comment]
@mhidas I have included your suggestions. Modification in imos_variable_test.nc are:

```
ggalibert@10-nsp-mel:~/compliance-checker$ ncDiff.sh /tmp/imos_variable_test.nc compliance_checker/tests/data/imos_variable_test.nc
32a33,34
>   byte bad3_qc ;
>       bad3_qc:quality_control_conventions = "unconventional" ;
55a58,59
> 
>  bad3_qc = -127 ;
```

**Comment by mhidas** [comment]
:+1:

**Pull request merged**